### ISSUE TITLE
[Enhancement] release 📦 Releases on Repository of Chocolatey packages

### ISSUE THREAD

**Issue by 4-FLOSS-Free-Libre-Open-Source-Software** [drventure/Disambiguator#3]
Hello, this is not an issue in itself. Just a suggestion. Would be nice to have an auto update feature for this primary security applications plugin from a trusted source. The SHA256 hash of PLRGX can be verified by installation automatically, optionally if specified.  _For the lazy ones and_ to fix incompatibility issue with future of KeePass version updates with plugin by getting all in one place for example.

I don't know if you have the spare time to maintain in parallel the "packages". Which basically would contain a script to URL of GitHub [latest](../releases/latest) release asset. 

_If you may want to take a look here if you don't know already:
[KeePass' plugins packages](https://community.chocolatey.org/packages?q=keepass-plugin&moderatorQueue=&moderationStatus=all-statuses&prerelease=true&sortOrder=package-download-count)
(There are many plugins and packages count is growing. )_

Another known existing autoupdate integration inside KeePass itself is:
[EarlyUpdateCheck KeePass Plugin](https://github.com/Rookiestyle/EarlyUpdateCheck#one-click-plugin-update) from which this Plugin is not supported yet.

**Comment by drventure** [comment]
Hi Bernd

That's a great idea, though I'm not terribly familiar with authoring Chocolatey packages.

The plugin itself is pretty simple and likely won't have any updates unless one becomes required because of KeePass changes.

What'd be great is if KeePass actually had a facility to apply updates instead of just checking if any are available. But the devs are likely a bit like me and only have a limited amount of free time.

By "auto update from a trusted source" I'm guessing you mean something like, 

If you see in Keepass that an update is available, then you'd just drop to a command line and same something like

Chocolatey install keepass-disambiguator

and it'd do the rest?

Darin

**Comment by drventure** [comment]
I've updated The Disambiguator to honor keepass's automatic update detection functionality and it appears to be working ok.

I'm going to go ahead and close this issue out, but feel free to reopen or post a comment if there's any other problems or comments on this.

**Comment by frankbits** [comment]
Wanted to add the plugin to [EarlyUpdateCheck](https://github.com/Rookiestyle/EarlyUpdateCheck).
Currently VERSION-File declares newest version  as 1.0.4 but latest [release](https://github.com/drventure/Disambiguator/releases) is 1.0.3. For this reason the Auto-Update can't work correctly.

Also the wrong latest Version is displayed in the KeePass-"Update Check" independent of if the EarlyUpdateCheck-plugin is used.

**Comment by drventure** [comment]
Thanks for the reminder. I'd done a few minor updates after 1.0.3 but got
sidetracked and never fully committed them.
Everything's updated and a new release 1.0.4 is now posted on Github.
